### ISSUE TITLE
BTCalpha not properly raising InsufficientBalance Error.

### ISSUE THREAD

**Issue by karlandoh** [ccxt/ccxt#5947]
- OS: Linux
- Programming Language version: Python 3
- CCXT version: Latest
- Exchange: ccxt.btcalpha()
- Method: ```createLimitOrder```

Good evening, would love for this exchange to properly capture insufficient balance errors. Instead it raises a different issue. Using TRX as an example.
```
btcalpha.createLimitBuyOrder('TRX/BTC',10000000,0.000001)

Request: POST https://btc-alpha.com/api/v1/order/ {'Accept': 'application/json', 'Content-Type': 'application/x-www-form-urlencoded', 'X-KEY': 'PERSONALLY OMITTED', 'X-SIGN': 'PERSONALLY OMITTED', 'X-NONCE': '1570599532120', 'User-Agent': 'python-requests/2.19.1', 'Accept-Encoding': 'gzip, deflate'} amount=10000000&pair=TRX_BTC&price=0.000001&type=buy

Response: POST https://btc-alpha.com/api/v1/order/ 201 {'Date': 'Wed, 09 Oct 2019 05:38:51 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '71', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=da6b8256884494690743dfa05e951244b1570599531; expires=Thu, 08-Oct-20 05:38:51 GMT; path=/; domain=.btc-alpha.com; HttpOnly; Secure', 'Vary': 'Accept, Accept-Language, Cookie', 'Allow': 'POST, OPTIONS', 'Content-Language': 'en', 'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload', 'X-Content-Type-Options': 'nosniff', 'X-Frame-Options': 'SAMEORIGIN', 'X-XSS-Protection': '1; mode=block', 'CF-Cache-Status': 'DYNAMIC', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '522e02bd2b24cacc-YYZ'} {"date":1570599531.4814300537,"error":"Out of balance -9.99243661 BTC"}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py", line 1719, in create_limit_buy_order
    return self.create_order(symbol, 'limit', 'buy', *args)
  File "/usr/local/lib/python3.7/site-packages/ccxt/btcalpha.py", line 307, in create_order
    if not response['success']:
KeyError: 'success'
```
Thanks for viewing.

**Comment by kroitor** [comment]
Should be ok as of CCXT 1.18.1235. Thx for posting a quality report!

**Comment by karlandoh** [comment]
Would love to make these changes myself to save you guys some time. Not exactly sure how to go about it, but I will get there. In the mean time, Ill try and post any additional issues. 

Thanks again

**Comment by kroitor** [comment]
@karlandoh 

> Would love to make these changes myself to save you guys some time. 

That would be awesome ) We really appreciate all the help from the community!

> Not exactly sure how to go about it (still a noob lol), Ive seen others post code in which you simply just 'merge' it.

The easiest way to get into it is to read the edits made to close each issue, for example, this particular issue was closed with commit 1dce705 and if you read the edits – the code is self-describing. So, you can gradually start seeing the structure behind the lib, start reading other merged PRs and discussions there and then [follow the rabbit](https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code) ;)
